src/js/yeldaWebchatInjector.js

```diff
@@ -95,7 +95,13 @@ class YeldaChat {
     // create assistantBubbleText element and set the text
     this.assistantBubbleText = document.createElement('span')
     this.assistantBubbleText.setAttribute('id', 'yelda_assistant_bubble_text')
-    this.assistantBubbleText.innerText = text || this.webchatSettings.bubbleText
+    this.assistantBubbleText.innerText = text
+    // Create close button and add it to the bubble text container
+    const closeButton = document.createElement('a')
```

Comment:
can we have tests for it ?
ex : 
- if, if `assistantBubbleText.innerText` is set, then close button exist
- on click on close button, the `yelda_assistant_bubble_text` element is removed
- on click on close button, the close button element is removed

why do we sometimes do ` this.assistantBubbleText = null` and sometime `this.removeElement('yelda_assistant_bubble_text')` (ex openChat)?